src/pybind/mgr/cephadm/upgrade.py

```diff
@@ -8,6 +8,7 @@
 from cephadm.serve import CephadmServe
 from cephadm.utils import name_to_config_section, CEPH_UPGRADE_ORDER
 from orchestrator import OrchestratorError, DaemonDescription
+from orchestrator._interface import daemon_type_to_service, service_to_daemon_types
```

Comment:
```suggestion
from orchestrator import daemon_type_to_service, service_to_daemon_types
```

Please don't import anything from `_interface` as this is a private python module. See

https://github.com/ceph/ceph/blob/3dd11ec576a99fb0452a4760e56b790693d7a18b/src/pybind/mgr/orchestrator/__init__.py#L10-L19